doc/src/manual/functions.md

```diff
@@ -364,9 +377,7 @@ julia> args = [1,2,3]
 
 julia> baz(args...)
 ERROR: MethodError: no method matching baz(::Int64, ::Int64, ::Int64)
-Closest candidates are:
```

Comment:
In the REPL this is the output:
```julia
julia> baz(args...)
ERROR: MethodError: no method matching baz(::Int64, ::Int64, ::Int64)
Closest candidates are:
  baz(::Any, ::Any) at REPL[1]:1
```

But the doctest does not run in the REPL so it will be something like:
```julia
julia> baz(args...)
ERROR: MethodError: no method matching baz(::Int64, ::Int64, ::Int64)
Closest candidates are:
  baz(::Any, ::Any) at none:1
```